docs/references/chrome-extension/create-clerk-client.mdx

```diff
@@ -3,131 +3,161 @@ title: '`createClerkClient()`'
+The `createClerkClient()` helper initializes a new Clerk instance on demand and refreshes the session token if there is a valid, signed-in user. It can be used in a [background service worker](https://developer.chrome.com/docs/extensions/develop/concepts/service-workers/basics) to access a user's information or token.
 
-When the extension popup is closed, the Clerk process that normally refreshes the user's session token every 60 seconds is no longer running. The stored session will become stale, as a closed popup or a background service worker do not run in the background.
+## Why use `createClerkClient()`?
 
-The `createClerkClient()` helper wille initialize a new Clerk instance on demand, and refresh the session token is there is a valid, signed user. This is the same process as returning to a web application after the tab or browser was closed. 
+It's recommended to use `createClerkClient()` for Chrome Extension's that are configured as popups. When the popup is closed, the Clerk process that normally refreshes the user's session token every 60 seconds is no longer running, and the stored session will become stale. If a request were made 60 seconds after the popup was closed, it would fail because the session token is no longer valid. By configuring `createClerkClient()` to run in a background service worker, you can ensure that the user's session is always fresh.
 
-This guide will demonstrate how to implement this in your Plasmo based Chrome Extension.
+## How it works
 
-Your application may need to interact with the background service worker from content injected into a webpage or from a new tab that your extension opens. For the sake of this guide you will create a web page that the extension will open as a new tab, and that page will have a button on it that retrieves the signed in user's token when clicked.
+A background service worker runs on demand. When a request comes in from a [content script](https://developer.chrome.com/docs/extensions/develop/concepts/content-scripts) that wants to interact with Clerk, the background service worker is triggered and calls `createClerkClient()`. This creates a Clerk instance and refreshes the session token.
 
-## Create your background service worker
+## Usage
 
-In your Plasmo based repo, create a new file called `index.ts` in `src/background`. In this file we will create a listener for messages from content script and create a function that uses `createClerkClient()` to get a new token for the user.
+The following example will demonstrate how to configure your Chrome Extension to use `createClerkClient()` to ensure that the user's session is always fresh. Your application may need to interact with the background service worker from content injected into a webpage or from a new tab that your extension opens. For this example, you will create a web page that the extension will open as a new tab, and that page will have a button on it that, when clicked, retrieves the signed in user's token.
```

Comment:
"...to ensure that the user's session is always fresh."

This isn't correct. The user's session can definitely be expired. This will allow the extension to instead valid and refresh the session on demand.

Reply:
"Your application may need to interact with the background service worker from content injected into a webpage or from a new tab that your extension opens."

While is a valid use case and is the example, this is probably not that common in reality. This example was just really easy to setup and really clear for the docs.

Reply:
I can remove the second part, but that was present in the document before so I kept that in so I didn't know it was incorrect.

```suggestion
The following example will demonstrate how to configure your Chrome Extension to use `createClerkClient()` to refresh your user's session token if it's valid. For this example, you will create a web page that the extension will open as a new tab, and that page will have a button on it that, when clicked, retrieves the signed in user's token.
```

Reply:
Yeah, that was my mistake.